tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
   "compilerOptions": {
-    "allowJs": false,
```

Comment:
Is this the default?

Reply:
Yes, it is the default. I basically removed it because there was a comment later in this file that suggested to remove it. I just tested that removing it, everything worked.